engine/access/state_stream/backend/handler.go

```diff
@@ -215,3 +217,260 @@ func (h *Handler) GetRegisterValues(_ context.Context, request *executiondata.Ge
 
 	return &executiondata.GetRegisterValuesResponse{Values: values}, nil
 }
+
+func (h *Handler) createAccountStatusesFilter(statusFilter *executiondata.StatusFilter) (state_stream.EventFilter, error) {
+	filter := state_stream.EventFilter{}
+	if statusFilter != nil {
+		var err error
+		filter, err = state_stream.NewEventFilter(
+			h.eventFilterConfig,
+			h.chain,
+			statusFilter.GetEventType(),
```

Comment:
I think we should maintain a list of events to include. Then if the users does not include any in their request, the default list is used. If the user does include a list, we match it against the default list and return an error if there's anything that's not listed.